### ISSUE TITLE
Ugly focus border around empty problems view

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#56064]
I see that the message in the problems view somehow has a tabindex assigned and looks like this when opening:

![image](https://user-images.githubusercontent.com/900690/43887190-5607a1ee-9bbe-11e8-8704-9fec1e323a8a.png)

I guess this might have been added as a fix for accessibility, but I think visually this really looks trashy and I also do not understand why this message should be accessible using tab key. If we want to announce something when you click on problems view, we should do it differently (alert?).

/cc @isidorn

**Comment by isidorn** [comment]
I also created this issue -> duplicate
